servicetalk-grpc-gradle-plugin/src/main/groovy/io/servicetalk/grpc/gradle/plugin/ServiceTalkGrpcPlugin.groovy

```diff
@@ -224,9 +220,6 @@ class ServiceTalkGrpcPlugin implements Plugin<Project> {
   }
 
   private static boolean createNewScriptFile(File outputFile) throws IOException {
-    if (outputFile.exists()) {
```

Comment:
can you clarify the motivation for this change? if the file already exists the assumption is we don't need to try to re-write the file. The script files are (or were before this PR) versioned so no need to re-write. If folks are using SNAPSHOT it is possible the contents have changed, but this will mostly be just for us during development and we can force a clean if we are working on the scripts.

Reply:
If we do not generate a new file on every run users can be in a situation when the previously generated file is not correct and contains path to the old `*-all.jar` that does not exist anymore. Examples:
1. gradle changes the hash for the cached file. Because the generated script contains the full path to the cached artifact, it won't run with the old one:
```
#!/bin/sh
exec java -jar ~/.gradle/caches/modules-2/files-2.1/io.servicetalk/servicetalk-grpc-protoc/0.26.0-SNAPSHOT/a2b7606f481385d75bc8fc419d6dfe244435a6b4/servicetalk-grpc-protoc-0.26.0-SNAPSHOT-all.jar "$@"
```
2. Users run with `--include-build`, then run the same project without `--include-build`. The script file contains value from the `serviceTalkProtocPluginPath` provided for `--include-build`. This makes script incorrect when users run it without `--include-build` later.

The overhead of writing a new file is very low. It will be cleaner and more reliable to override the old file on every run.